src/AppInstallerCLIE2ETests/ListCommand.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace AppInstallerCLIE2ETests
+{
+    using NUnit.Framework;
+
+    public class ListCommand
+    {
+        private const string ListTestSourceName = @"ListTestSource";
+        private const string ListTestSourceUrl = @"https://localhost:5001/TestKit";
+
+        [SetUp]
+        public void Setup()
+        {
+            Assert.AreEqual(Constants.ErrorCode.S_OK, TestCommon.RunAICLICommand("source add", $"{ListTestSourceName} {ListTestSourceUrl}").ExitCode);
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            TestCommon.RunAICLICommand("source remove", ListTestSourceName);
+            TestCommon.WaitForDeploymentFinish();
+        }
+
+        //[Test]
+        public void ListTest()
+        {
+            // Example List Command Test
+            var installDir = TestCommon.GetRandomTestDir();
+            TestCommon.RunAICLICommand("install", $"AppInstallerTest.TestExeInstaller --silent -l {installDir}");
+            var result = TestCommon.RunAICLICommand("list", "");

```

Comment:
>list [](start = 53, length = 4)

List is supposed to show all installed apps (including those that weren't installed via WinGet) - Correct?

Reply:
Yes, you are correct. Added additional test to cover installed apps not installed via WinGet.